pyproject.toml

```diff
@@ -23,11 +23,11 @@ repository="https://github.com/tokern/dbcat/"
 
 
 [tool.poetry.dependencies]
-python = ">=3.6.1,<3.9"
+python = ">=3.7,<=3.10.8"
```

Comment:
Should this be `>=3.8` since the tests run only for 3.8 onwards?

Reply:
👍